tools/build-tools/.vscode/launch.json

```diff
@@ -13,6 +13,19 @@
                 "${workspaceFolder}/**/*.js"
             ]
         },
+        {
```

Comment:
Do we have docs on this process that we should add a note to about this addition?

Reply:
Do you mean the process of adding a new debug config, or the process of bumping versions? The docs for the release process will be on eng.ms.

Reply:
Sorry - yeah, I meant bumping process docs. Wondering if we should call out the vscode option in those docs. I was mostly thinking if there were any process docs in a README or something, then it would be worth updating. But if the docs are elsewhere, then that falls outside the scope of the PR, so I'll resolve this :)